website/content/docs/release-notes/consul/v1_15_x.mdx

```diff
@@ -0,0 +1,65 @@
+
+- **Consul Envoy Extensions:** A new extension system has been added to Consul to modify Consul-generated Envoy resources outside of the Consul binary. This will allow extensions to add, delete, and modify Envoy listeners, routes, clusters, and endpoints, enabling support for additional Envoy features without changes to the Consul codebase.
+Current supported extensions include the [Lua](https://developer.hashicorp.com/consul/docs/connect/proxies/envoy-extensions#lua) and [AWS Lambda](https://developer.hashicorp.com/consul/docs/connect/proxies/envoy-extensions#lambda) extensions. Refer to [Envoy extensions overview](https://developer.hashicorp.com/consul/docs/connect/proxies/envoy-extensions) for more information on how to leverage these extensions for Consul service mesh.  
+
+- **API Gateway support on Linux VM runtimes:** The Consul API Gateway can now be deployed on Linux VM runtimes. API Gateway is built into Consul and, when deploying on Linux VM runtimes, is not separately installed software.
+
+  ~> **Note:** Support for API Gateway on Linux VM runtimes is considered a ”Beta” feature in Consul 1.15.0. HashiCorp expects to change it to a GA feature as part of a 1.15 patch release in the near future.
+
+- **Service-to-service troubleshooting:** Consul now includes a built-in tool for troubleshooting communication between services in a service mesh. A new `consul troubleshoot` command is provided to validate communication between upstream and downstream Envoy proxies on VM and Kubernetes deployments.
+Refer to [Service-to-service troubleshooting overview](https://developer.hashicorp.com/consul/docs/troubleshoot/troubleshoot-services) for more details on how to use the new troubleshooting commands. 
+
+- **Raft write-ahead log (Experimental):** Consul now provides an experimental storage backend called write-ahead log (WAL). WAL implements a traditional log with rotating, append-only log files which resolves a number of performance issues with the current BoltDB storage backend. Refer to [Experimental WAL LogStore backend overview](https://developer.hashicorp.com/consul/docs/agent/wal-logstore) for more details.   
+  
+  ~> **Note:** The new raft write-ahead log storage backend is not recommended for production use cases yet, but is ready for testing by the general community. 
+
+## What's Changed
+
+- ACL errors have now been ehanced to return descriptive errors when the specified resource cannot be found. Other ACL request errors provide more information about when a resource is missing. Add error for when the ACL system has not been bootstrapped.
+  - The Delete Token/Policy/AuthMethod/Role/BindingRule endpoints now return 404 when the resource cannot be found.
+    - The new error format is as follows: `Requested * does not exist: ACL not found", "* not found in namespace $NAMESPACE: ACL not found`
```

Comment:
I think this format might look better. Can you change the others to reflect this? Might be nice to link to the respective API docs here too

````suggestion
  - The Delete Token/Policy/AuthMethod/Role/BindingRule endpoints now return 404 when the resource cannot be found. The new error format is as follows:

    ```log hideClipboard
    Requested * does not exist: ACL not found", "* not found in namespace $NAMESPACE: ACL not found`
    ```

````